packages/sdk/identity/package.json

```diff
@@ -44,7 +44,8 @@
     "fetch-mock": "9.10.4",
     "@types/elliptic": "^6.4.12",
     "@celo/flake-tracker": "0.0.1-dev",
-    "@celo/ganache-cli": "git+https://github.com/celo-org/ganache-cli.git#21652da"
+    "@celo/ganache-cli": "git+https://github.com/celo-org/ganache-cli.git#21652da",
+    "@celo/identity": "1.5.2"
```

Comment:
ah it depends on itself?

Reply:
oh for the test

Reply:
hmm it would probably be smart to alias this

Reply:
+1 this should be aliased https://www.dev-diaries.com/social-posts/install-npm-package-under-alias/

Reply:
good suggestion thanks!